### PR TITLE
Fix/update nf tests

### PR DESCRIPTION
**Pull request by sateeshperi**
Updating nf-tests to include each version of test in the snapshot

## PR checklist

- [ ] This comment contains a description of changes (with reason).
- [ ] If you've fixed a bug or added code that should be tested, add tests!
- [ ] If you've added a new tool - have you followed the pipeline conventions in the [contribution docs](https://github.com/nf-core/methylseq/tree/master/.github/CONTRIBUTING.md)- [ ] If necessary, also make a PR on the nf-core/methylseq _branch_ on the [nf-core/test-datasets](https://github.com/nf-core/test-datasets) repository.
- [ ] Make sure your code lints (`nf-core lint`).
- [ ] Ensure the test suite passes (`nextflow run . -profile test,docker --outdir <OUTDIR>`).
- [ ] Usage Documentation in `docs/usage.md` is updated.
- [ ] Output Documentation in `docs/output.md` is updated.
- [ ] `CHANGELOG.md` is updated.
- [ ] `README.md` is updated (including new tool citations and authors/contributors).

### CODE REVIEW

**Comment by nvnieuwk** [comment]
I think using [`assertAll`](https://code.askimed.com/nf-test/assertions/assertions/#using-assertall) should be standard practice for us. Also you can check lines of files in snapshots too instead of the whole md5sum :)

**Comment by sateeshperi** [comment]
good point...i ll enclose them in assertAll

File: tests/bismark/main.nf.test [hunk 1]
```diff
@@ -24,19 +24,22 @@ nextflow_pipeline {
...
-            // FIXME assert snapshot(workflow).match()
-            assert new File("$outputDir/multiqc/bismark/multiqc_report.html").exists()
 
-            assert new File("$outputDir/bismark/methylation_calls/splitting_report/SRR389222_sub1_trimmed_bismark_bt2.deduplicated_splitting_report.txt").exists()
-            assert new File("$outputDir/bismark/methylation_calls/splitting_report/SRR389222_sub2_trimmed_bismark_bt2.deduplicated_splitting_report.txt").exists()
-            assert path("$outputDir/bismark/methylation_calls/splitting_report/SRR389222_sub2_trimmed_bismark_bt2.deduplicated_splitting_report.txt").md5 == "3038a90a6e513c3b1c0de540439eb101"
+            // Snapshot
+            assert snapshot(path("$outputDir/bismark/methylation_calls/splitting_report/Ecoli_10K_methylated_1_val_1_bismark_bt2_pe.deduplicated_splitting_report.txt"),
+                            path("$outputDir/bismark/methylation_calls/splitting_report/SRR389222_sub1_trimmed_bismark_bt2.deduplicated_splitting_report.txt"),
+                            path("$outputDir/bismark/methylation_calls/splitting_report/SRR389222_sub2_trimmed_bismark_bt2.deduplicated_splitting_report.txt"),
```
Comment (Emiller88) [inline tests/bismark/main.nf.test:33]: Are these reproducible? I don't think I could get a consistant md5sum out of them. Or is there some way to turn the md5sum off in the snapshot?

**Review by Emiller88 — COMMENTED** [review]
Left some thoughts

File: tests/bismark/main.nf.test (hunk 1 above)
Comment (sateeshperi) [reply tests/bismark/main.nf.test:31]: i am getting consistent md5s for these files. Could you give it a test @Emiller88 ?

**Comment by ewels** [comment]
Let's update the template and all modules before merging this, as that will likely modify test outputs: https://github.com/nf-core/methylseq/pull/293

File: tests/bismark/main.nf.test (hunk 1 above)
Comment (Emiller88) [reply tests/bismark/main.nf.test:31]: If you and the CI are passing then I think I'd be the odd one out anyway, I think it's good then!

**Comment by ewels** [comment]
Or, if you prefer - change the base to branch `nf-core-template-merge-2.7.2` and update, so that so that we can merge into PR #293 and then merge both together with passing tests ✅ 

Might be the easiest way to go..

**Pull request closed without merging**